meson.build

```diff
@@ -1,4 +1,4 @@
-project('rizin', 'c', license: 'LGPL3', meson_version: '>=0.55.0')
+project('rizin', 'c', license: 'LGPL3', meson_version: '>=0.50.0', default_options: ['c_std=c99'])
```

Comment:
no please, keep 0.55.0, because there are things we use that require at least 0.54.something and we are going to switch to meson subprojects soon, which require  0.55.0 anyway.

Reply:
oups, wrong merge, sorry